dotnet/src/IntegrationTests/Connectors/Memory/AzureAISearch/AzureAISearchConfigSetConditionAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using SemanticKernel.IntegrationTests.Connectors.Memory.Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Memory.AzureAISearch;
+
+/// <summary>
+/// Attribute to use to skip tests if the settings for Azure AI Search is not set.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
+public sealed class AzureAISearchConfigSetConditionAttribute : Attribute, ITestCondition
+{
+    public ValueTask<bool> IsMetAsync()
```

Comment:
nit: Just a small renaming suggestion:

```suggestion
public sealed class AzureAISearchConfigConditionAttribute : Attribute, ITestCondition
{
    public ValueTask<bool> IsValidAsync()
```

Reply:
Sure, attribute is easy to rename right now, but method name will have a cascade all over the place.  I can do the method name separately.